internals/scripts/CheckNativeDep.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+import fs from 'fs';
+import chalk from 'chalk';
+
+const EndOfLine = require('os').EOL;
+const { execSync } = require('child_process');
```

Comment:
Let's use ES6 imports for all of these imports.

Reply:
Fixed